src/opossum_lib/attribution_generation.py

```diff
@@ -9,13 +11,21 @@
 from opossum_lib.opossum_file import OpossumPackage, SourceInfo
 
 
+def _get_purl(package: Package) -> Optional[str]:
+    for external_reference in package.external_references:
+        if external_reference.reference_type == "purl":
```

Comment:
I'd suggest to extract the string `"purl"` into a constant

Reply:
done